src/plays/react-transitions/TransitionSearch.jsx

```diff
@@ -36,8 +36,8 @@ export default function TrainsitionSearch({ users }) {
       ) : (
         <div className="user-cards">
           {filtered.length > 0 ? (
-            filtered.map((user) => (
-              <div class="card">
+            filtered.map((user, index) => (
+              <div className="card" key={index}>
```

Comment:
Same here

Reply:
Hi all. Sorry for the late reply.
For this play, when you will type for a new search entry, all the results will change and the result will change each time with the input. 
You didn't remove/add an element from a list or something like that, but just implement a filter. So, it didn't impact the code to use the index or name, or other attribute.

uuid is a good choice but you need to use it only when you really need it ( more weight in the  final code ) but can be use to have a way to create id every where in the code base in the same way.